osama-breakout-room/zeena_kareem/style.css

```diff
@@ -0,0 +1,123 @@
+*{
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+  
+}
+.container{
+    background: rgb(231, 222, 222);
+    box-shadow:0px 0px 5px 0px gray ;
+}
+.container_image{
+    margin-left:25%;
```

Comment:
I don't think its really good practice to use % with margins but rather use px instead